Dockerfile

```diff
@@ -1,32 +1,59 @@
-#FROM python:3.12.2
-FROM python:3.12.2-slim-bookworm
+FROM ubuntu
+# This will not work, since debian uses a slightly different apache2 config structure.
+# Current Dockerfile is meant to be used with the ubuntu apache2 setup.
+# FROM python:3.12.2-slim-bookworm
 
-ENV PYTHONUNBUFFERED 1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
 
-# Set the working directory in the container
-WORKDIR /app
+# Make port 80 and 443 available to the world outside this container.
+# 80 will be redirected to 443 using TLS through the apache.
+EXPOSE 80 443
+
+RUN apt-get update && apt-get install -y apt-utils vim curl apache2 apache2-utils python3 libapache2-mod-wsgi-py3 python3-pip python3-venv
```

Comment:
I would remove unnecessary packages like vim (maybe curl as well? IDK), so the image stays slim and therefor minimize potential security vulnerabilities.